api/src/Model/Model/IndexedModel.php

```diff
@@ -70,15 +79,25 @@ public function countIndex()
 
     public function getIndexDocumentData()
     {
-        $modelArray = $this->toArray();
-
-        foreach ($modelArray as $attribute => &$value) {
-            if ($value instanceof Carbon) {
-                $value = $value->toIso8601String();
+        $relations = [];
+
+        if (! empty($this->indexNested)) {
```

Comment:
I wonder for this if we should be actually using the transformer service, as it does pretty much this, and it does the formatting of the timestamps into the correct format for elastic (iso8601)